### DISCUSSION TITLE
guildMemberAdd event not getting called even when intents enabled from developer portal and client intents as well when new user joins

### DISCUSSION THREAD

**Discussion by neurlnet** [discordjs/discord.js#9557] [Q&A] [answered]
Which package is this bug report for?
discord.js

Issue description
I have provided all the necessary intents from both the developer portal as well as in the code. Still the guildMemberAdd is not getting called, I have refreshed the bot multiple times, I have testing in different servers with different users, still the issue persists, I have checked all other similar issues, but the fixes stated adding the intents, which I have already applied but still the issue persists.

Any help would be highly appreciated!

Code sample
const { Client, GatewayIntentBits, Collection } = require(`discord.js`);
const client = new Client({ intents: [GatewayIntentBits.GuildVoiceStates, GatewayIntentBits.Guilds, GatewayIntentBits.GuildMessages, GatewayIntentBits.MessageContent, GatewayIntentBits.GuildMessageReactions, GatewayIntentBits.GuildMembers, GatewayIntentBits.GuildPresences] });

client.on('guildMemberAdd', (guildMember) => {
    console.log("A new member just joined some server!")
})

Versions
discord.js v14.11.0
Node.js v16.9.0
Issue priority
Medium (should be fixed soon)

Which gateway intents are you subscribing to?
Guilds, GuildMembers, GuildVoiceStates, GuildPresences, GuildMessages, GuildMessageReactions, MessageContent

**Comment by neurlnet** [comment]
nevermind, It was a issue on my hosting side

**Comment by almostSouji** [comment, marked as answer]
This was likely not related to your host or user- or library error, but an instance of a Discord API issue: https://discordstatus.com/incidents/6xgts6fdwnr7 which has now been resolved.

**Comment by neurlnet** [comment]
oh thanks
